images/installer/Dockerfile.upi.ci

```diff
@@ -23,7 +23,7 @@ RUN yum install --setopt=tsflags=nodocs -y \
     openssh-clients && \
     yum update -y && \
     yum install --setopt=tsflags=nodocs -y \
-    unzip gzip jq awscli util-linux && \
+    unzip gzip awscli util-linux && \
```

Comment:
Can you provide a reason why we are dropping jq ?

Reply:
CI can't install epel-release and has an issue with internal repos, so upi-installer build fails with it.

@stevekuznetsov is looking at it afaik

Reply:
@smarterclayton where do the `devel-4.0` rpm repos come from?

Reply:
Builds for this started to see them unreachable